blockchain_connector/Dockerfile

```diff
@@ -120,53 +117,157 @@ RUN mkdir -p build \
   && cmake .. -DUNTRUSTED_ONLY=1 \
   && make
 
-#install avalon common module
+
+# -------------=== common/python build ===-------------
+
+#Build common/python intermediate docker image
+FROM python_image as common_python_image
+
+RUN apt-get update \
+ && apt-get install -y -q \
+    make
+
+# Copy openssl build artifacts from openssl_image
+COPY --from=openssl_image /usr/local/ssl /usr/local/ssl
+COPY --from=openssl_image /usr/local/bin /usr/local/bin
+COPY --from=openssl_image /usr/local/include /usr/local/include
+COPY --from=openssl_image /usr/local/lib /usr/local/lib
```

Comment:
My understanding is that Avalon common python doesn't depend on openssl now. @manju956 can you confirm. If so, we can remove above lines

Reply:
That's right, common python don't depend on openssl as crypto was separated out of common python.